module/help_mode.c

```diff
@@ -1,4 +1,4 @@
-#include "help_mode.h"
+#include "help_mode.h"1
```

Comment:
a stray `1` at the end of the line here?

Reply:
Huh! Seems odd that this did not get caught by CI?

Reply:
yeah, strange!